### PR TITLE
Update Plugin.rst - Add persistence.recursive

### PR DESCRIPTION
**Pull request by Tim-Obert**

### CODE REVIEW

File: Documentation/TopLevelObjects/Plugin.rst [hunk 1]
```diff
@@ -215,6 +215,34 @@ persistence.storagePid
...
+.. container:: table-row
+
+   Property
+      persistence.recursive
+
+   Data type
+      :ref:`data-type-integer`
+
+   Description
+      **Only for Extbase plugins**. Number of how many sub-levels of the
```
Comment (sarahmccarthy123) [inline Documentation/TopLevelObjects/Plugin.rst:232]:
```suggestion
**Only for Extbase plugins**. Number of sub-levels of the 
```

**Review by sarahmccarthy123 — DISMISSED** [review]

File: Documentation/TopLevelObjects/Plugin.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/TopLevelObjects/Plugin.rst:232]: @sarahmccarthy123 Please add the indent at the beginning again.

Comment (sarahmccarthy123) [reply Documentation/TopLevelObjects/Plugin.rst:232]:
```suggestion
      **Only for Extbase plugins**. Number of sub-levels of the 
```

Comment (brotkrueml) [reply Documentation/TopLevelObjects/Plugin.rst]: @sarahmccarthy123 I can't commit the suggestion, can you?

**Review by sarahmccarthy123 — APPROVED** [review]

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@Tim-Obert Thanks for the improvement :+1:

**Pull request merged**